browser/src/layer/marker/TextInput.js

```diff
@@ -110,6 +104,18 @@ window.L.TextInput = window.L.Layer.extend({
 
 	},
 
+	_addCursorHandler() {
+		if (document.getElementById('canvas-container')) {
+			this._cursorHandler = new CursorHandler();
+			app.sectionContainer.addSection(this._cursorHandler);
+		}
+		else {
+			setTimeout(() => {
+				this._addCursorHandler();
+			}, 200);
```

Comment:
can we do it more deterministic? can't it be a layoutingTask?

`app.layoutingService.appendLayoutingTask()` which will be executed after all DOM modifications
the problem we had with animations in the past was that DOM recalculations happened due to DOM access in random moments